api/facadeversions.go

```diff
@@ -23,7 +23,7 @@ var facadeVersions = map[string]int{
 	"ApplicationScaler":            1,
 	"Backups":                      2,
 	"Block":                        2,
-	"Bundle":                       1,
+	"Bundle":                       2,
```

Comment:
Until we have the implementation finished, we don't want to expose this to clients, so keep this at 1 for now.